### DISCUSSION TITLE
How about re-rerender all the theme images with a standard screen of code at 4k resolution?

### DISCUSSION THREAD

**Discussion by David-Else** [helix-editor/helix#2227] [Ideas]
https://github.com/helix-editor/helix/wiki/Themes has a mix of different resolutions, font size, framing of the screen, and image quality. On 4k they nearly all look awful! It is not a good advert for the awesome themes.

Someone could decide on a classic bit of code (maybe JS as it is the most common?) that could be used to create an image for each theme, then it would be easy to tab through each theme and take a screenshot.

It should probably wait until the next official version of Helix is released so all the new themes are included.

Also, it would be nice to have them on the main website on a nice slider so people can browse them with ease :)

**Comment by kirawi** [comment]
Is there a way to do that? Maybe something like asciicinema's terminal emulator?

**Reply by workingj** [reply]
I think it could be done via a bash script in changing and reloading the config, and taking and renaming the screenshot while iterating over the list of theme names.

Maybe binding it to a key for repeating the process from within helix....

**Reply by valpackett** [reply]
Adding a way to capture an interactive application to https://github.com/homeport/termshot might be a good idea…
